modules/core/02-client/client/cli/tx.go

```diff
@@ -461,3 +457,96 @@ func NewCmdSubmitUpgradeProposal() *cobra.Command {
 
 	return cmd
 }
+
+// newCmdScheduleIBCUpgradeProposal defines the command for submitting an IBC software upgrade proposal.
+func newCmdScheduleIBCUpgradeProposal() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "schedule-ibc-upgrade [name] [height] [path/to/upgraded_client_state.json] [flags]",
+		Args:  cobra.ExactArgs(3),
+		Short: "Submit an IBC software upgrade proposal",
+		Long: "Please specify a unique name and height for the upgrade to take effect.\n" +
+			"The client state specified is the upgraded client state representing the upgraded chain\n" +
```

Comment:
seems odd to have the `Long` string consist of multiple regular quoted strings with new lines, and then also a template string, I think we could just do
```golang
Long: `Please specify a unique name and height for the upgrade to take effect.
The client state specified is the upgraded client state representing the upgraded chain

Example Upgraded Client State JSON: 
// example
`
```